Cargo.toml

```diff
@@ -82,6 +86,16 @@ serde = [
     "ipnet/serde",
 ]
 
+# Internal use only
```

Comment:
Mostly trying to stay DRY here, but happy to manually copy and paste these across both the `parser` and `parser-rustls` feature flags.